Tools/CMake/AMReXThirdPartyLibraries.cmake

```
@@ -3,13 +3,13 @@
 #
 if (ENABLE_HDF5)
     set(HDF5_PREFER_PARALLEL TRUE)
-    find_package(HDF5 1.10.4 REQUIRED COMPONENTS CXX)
+    find_package(HDF5 1.10.4 REQUIRED)
```

I would go with and explicit request of what we need:
```suggestion
    find_package(HDF5 1.10.4 REQUIRED COMPONENTS C)
```

---

From the [FindHDF5 doc](https://cmake.org/cmake/help/v3.14/module/FindHDF5.html): ```he module will optionally accept the COMPONENTS argument. If no COMPONENTS are specified, then the find module will default to finding only the HDF5 C library. ```

---

Sure, unless this behavior changes in the future and/or a superbuild has an unexpected environment set.